Commands/wombat.py

```diff
@@ -0,0 +1,23 @@
+import os
+import discord
+import random
+
+from discord.ext import commands
+
+class Wombat(commands.Cog, name="Wombat"):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="wombat_pic", brief="Post a random wombat picture", usage="", help="The bot will search in its database for a random piccture of a wombat, and post it in the chat")
+    async def sub(self, ctx):
```

Comment:
Change this name to something else than sub?

Reply:
would suggest `cmd_wombat_pic`
I name all my commands with prefix `cmd_` to separate them from other methods

Reply:
I like this idea 👍